### PR TITLE
Add instructions on making the shell fullscreen and fixy fixy some stuffy

### PR DESCRIPTION
**Pull request by halal-beef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,12 @@ Rename image-new.img to boot.img, then using 7zip (on windows) or tar on linux,
 
 You can now flash boot.tar onto your phone using ODIN3, Heimdall, or TWRP if you prefer.
 
+### Getting EDK2 shell fullscreen
+
+If you made it this far, congratulations but you're here to get the shell fullscreen not to grt some kind of trophy. To make the shell fullscreen open up EXYNOS7885Pkg/Drivers/GraphicsConsoleDxe/GraphicsConsole.c, go to line 293
```
Comment (leandrofriedrich) [inline README.md]: *get
and some missign kommata

**Review by leandrofriedrich — APPROVED** [review]
Ballin

File: README.md (hunk 1 above)
Comment (halal-beef) [reply README.md]: Outdated u fucker, I fixed that ages ago

**Comment by sonic011gamer** [comment]
K, thanks for the mini guide :trollface:

**Pull request merged**